src/cogent3/util/warning.py

```diff
@@ -105,7 +109,7 @@ def deprecated_args(
     Here's an example of how to use the `deprecated_args` decorator to mark the argument `old_name` as deprecated
     and replace it with the new name `new_name`.
 
-    >>> @deprecated_args(mapping=[('old_name', 'new_name')], version='2.0', reason='Use new_name instead')
+    >>> @deprecated_args(mapping=[('old_name', 'new_name')], version='2.0', reason='Use new_name instead', is_discontinues=True)
```

Comment:
typo, should be `is_discontinued`

Reply:
OK will make those changes and also clean up testing (similar to the way you did your tests).  I'm going to extract out a function to nominate a func ('method' or 'function'  + name) as I should use the same convention for args.

Reply:
I decided to call the utility functions directly rather than messing with custom params.  Pretty happy with the UX from a developer POV right now.  

Check out the chained decorators test.